pkg/capacity/features.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package capacity
+
+import (
+	"fmt"
+	"strings"
+
+	flag "github.com/spf13/pflag"
+)
+
+// Feature is the type for named features supported by the capacity
+// controller.
+type Feature string
+
+// Features are disabled by default.
+type Features map[Feature]bool
+
+const (
+	// FeatureCentral enables the mode where there is only one
+	// external-provisioner actively running in the cluster which
+	// talks to the CSI driver's controller.
```

Comment:
add "service" to be more explicit we're talking about the csi service

Reply:
Done.